decidim-assemblies/app/models/decidim/assembly_member.rb

```diff
@@ -13,7 +13,7 @@ class AssemblyMember < ApplicationRecord
     belongs_to :assembly, foreign_key: "decidim_assembly_id", class_name: "Decidim::Assembly"
     alias participatory_space assembly
 
-    default_scope { order(arel_table[:weight].asc) }
+    default_scope { order(weight: :asc, created_at: :asc) }
```

Comment:
Maybe we should update the indexes at the DB

Reply:
Done!